src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -50,7 +50,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
-    String aboutMe = Jsoup.clean(request.getParameter("about-me"), Whitelist.none());
+    // filters user inputted about me for safe html
```

Comment:
nit: Capitalize the first letter of your comment and comment should be more specific to what you are doing. Filters is too ambiguous as it can me include or exclude. In this case, you are including safe html and excluding bad html.

"// Whitelists safe html user input in the about me text box."